library/core/src/intrinsics.rs

```diff
@@ -1929,11 +1929,10 @@ pub const unsafe fn copy_nonoverlapping<T>(src: *const T, dst: *mut T, count: us
 /// ```
 #[doc(alias = "memmove")]
 #[stable(feature = "rust1", since = "1.0.0")]
-#[rustc_const_unstable(feature = "const_intrinsic_copy", issue = "none")]
+#[rustc_const_unstable(feature = "const_intrinsic_copy", issue = "80697")]
 #[inline]
 pub const unsafe fn copy<T>(src: *const T, dst: *mut T, count: usize) {
     extern "rust-intrinsic" {
-        #[rustc_const_unstable(feature = "const_intrinsic_copy", issue = "none")]
```

Comment:
This is not needed, right?

Reply:
I would have thought it is, otherwise you should not be able to call it below, looks like we have a hole in our analysis somewhere. The `rustc_const_unstable` attribute is what makes the intrinsic `const`, since we cannot have `const fn` intrinsics since the syntax doesn't allow this.

Reply:
uff.. I just checked, we have no unit tests ensuring that this actually works, so it must've regressed.

Reply:
The reason I noticed it was because I did not see the corresponding line for `copy_nonoverlapping`

Reply:
I believe I got the error before the rebase(the reason the attribute was on `copy` at all)

Reply:
As far as I can tell, only looking at #79684, the error must have disappeared somewhere between the 4th and 21th of December. Quite a large span, but perhaps somewhere tool start...

Reply:
I... have an inlkling. I'm waiting for a rebuild with some more `trace!` instructions, now that I am able to reproduce. I think the issue may be that the `copy` intrinsic is declare *inside* a function with `rustc_const_unstable` and *inherits* that